docs/markdown/Helm/helm-overview.md

```diff
@@ -253,6 +255,20 @@ tests:
 > not affected by the source roots setting. When using `relocated_files`, the files will be relative
 > to the value set in the `dest` field.
 
+### Snapshot testing
+
+Using unit test snapshots is fully supported by Pants as test snapshots are just resources as shown in the previous section.
+
+However, since managing snapshots by hand is quite tedious, Pants gives some utilities to manage them in a simpler way. To automate the update of the snapshots use Pants' `generate-snapshots` goal which we can use to generate the snapshot files:
+
+```
+pants --filter-target-type='helm_unittest_test' generate-snapshots ::
+```
+
+That would generate the test snapshots for those tests that require it with older snapshots being overwritten and newer ones being created.
+
+If new `__snapshot__` folders have been created we recommend to run again the `tailor` goal and Pants would recognise those new folders and create `resource` targets as appropriate.
```

Comment:
```suggestion
If new `__snapshot__` folders are created we recommend running the `tailor` goal, so that Pants can detect these new folders and create `resource` targets as appropriate.
```